tests/cloudpickle_test.py

```diff
@@ -2235,6 +2235,10 @@ def test_unhashable_function(self):
         self.assertEqual(depickled_method("a"), 1)
         self.assertEqual(depickled_method("b"), None)
 
+    @unittest.skipIf(
+        sys.version_info >= (3, 14),
+        "itertools.count() doesn't support pickle on Python 3.14+",
+    )
```

Comment:
@vstinner isn't that considered a regression? Why wouldn't such a simple object not be picklable?